spec.emu

```diff
@@ -20,54 +20,138 @@ contributors: Ujjwal Sharma, Younies Mahmoud
+          1. Set _stylesList_ to the list-concatenation of _stylesList_ and &laquo; "numeric" &raquo;.
+          1. If _unit_ is one of `"hours"`, `"minutes"`, or `"seconds"`, then
+            1. Set _stylesList_ to the list-concatenation of _stylesList_ and &laquo; "2-digit" &raquo;.
+        1. Let _style_ be ? GetOption(_options_, _unit_, _stylesList_, *undefined*).
+        1. If _style_ is *undefined*, then
+          1. If _baseStyle_ is `"digital"`, then
+            1. If _unit_ is one of `"hours"`, `"minutes"`, `"seconds"`, `"milliseconds"`, `"microseconds"`, or `"nanoseconds"`, then
+              1. Set _style_ to `"numeric"`.
+            1. Else,
+              1. Set _style_ to `"narrow"`.
+          1. Else,
+            1. Set _style_ to _baseStyle_.
+          1. Let _displayDefault_ to `"auto"`.
+        1. Else,
+          1. Let _displayDefault_ to `"always"`.
+        1. Let _displayField_ be the string-concatenation of _unit_ and `"Display"`.
+        1. Let _display_ be ? GetOption(_options_, _displayField_, &laquo; `"auto"`, `"always"` &raquo;, _displayDefault_).
+        1. If _prevStyle_ is `"numeric` or `"2-digit`, then
+          1. If _style_ is not `"numeric` or `"2-digit"`, then
+            1. Throw a *RangeError* exception.
+          1. Else if _unit_ is `"minutes"` or `"seconds"`, then
+            1. Set _style_ to `"2-digit"`.
+        1. Return the Record {
+          [[Style]]: _style_,
+          [[Display]]: _display_
+        }.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-clipduration" aoid="ClipDuration">
+      <h1>ClipDuration ( _duration_, _upper_, _lower_ )</h1>
+      <p>The ClipDuration abstract operation is called with the arguments _duration_ (which must be an object initialized as a Temporal.Duration), _upper- (which must be a string), and _lower_ (which must be a string), and returns the _duration_ clipped at between the specified units. The following steps are taken:</p>
+      <emu-alg>
+        1. Let _clipped_ be ? CreateTemporalDuration(0, 0, 0, 0, 0, 0, 0, 0, 0, 0).
+        1. For each row in <emu-xref href="#table-duration-components"></emu-xref>, starting from _unit_, in table order, do
+          1. If _unit_ is _lower_, then
+            1. Return _clipped_.
+          1. Let _valueSlot_ be the Value Slot property.
+          1. Let _value_ be _duration_.[[&lt;_valueSlot_&gt;]].
+          1. Set _clipped_.[[&lt;_valueSlot_&gt;]] to _value_.
+        1. Return _clipped_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-numericlimit" aoid="NumericLimit">
+      <h1>NumericLimit (_durationFormat_, _start_)</h1>
+      <p>The NumericLimit abstract operation is called with the arguments _duration_ (which must be an object initialized as a DurationFormat), and _start_ (which must be a string), and returns a string. The following steps are taken:</p>
```

Comment:
The purpose of this AO isn't immediately obvious (I suspect it's to support figuring out how to display fractional numbers?) so it'd be nice to have a sentence here explaining the purpose of this AO.

Reply:
This is no longer needed, removing.